src/types/log.ts

```diff
@@ -0,0 +1,77 @@
+// Transaction
+import { JsonRpcTx } from "../deps.ts";
+
+// Starknet
+import { Event, hash } from "../deps.ts";
+
+// Eth
+import { bigIntToHex } from "../deps.ts";
+
+const IGNORED_KEYS = [
+  hash.getSelectorFromName("transaction_executed"),
+  hash.getSelectorFromName("evm_contract_deployed"),
+  hash.getSelectorFromName("Transfer"),
+  hash.getSelectorFromName("Approval"),
+];
+
+/**
+ * @param transaction A Ethereum transaction.
+ * @param event A Starknet event.
+ * @returns The log in the Ethereum format, or null if the log is invalid.
+ */
+export function toEthLog(
+  { transaction, event }: {
+    transaction: JsonRpcTx;
+    event: Event;
+  },
+): JsonRpcLog | null {
+  // Filter out ignored events which aren't ETH logs.
+  if (IGNORED_KEYS.includes(event.keys[0])) {
+    return null;
+  }
+
+  if (event.keys.length < 1 || event.keys.length % 2 !== 1) {
+    console.error(`Invalid event ${event}`);
+    return null;
+  }
+
+  // The address is the first key of the event.
+  const address = bigIntToHex(BigInt(event.keys[0]));
+  const data = event.data.map((d) => BigInt(d).toString(16).padStart(2, "0"))
```

Comment:
can we add a comment that shows an example of this small trick or explains the logic